File: Tareas/Unidad2/a2253332351_practica01/09_MontoCapital.cpp
#include <iostream>
using namespace std;

int main() {
    double capital;
    double interes;
    double monto;

    cout << "Ingrese el capital: ";
    cin >> capital;
    cout << "Ingrese la tasa de interés: ";
    cin >> interes;

    monto = capital * (1 + (interes / 100));

    cout << "El monto es: " << monto << endl;

    return 0;
}

File: Tareas/Unidad2/a2253332351_practica01/09_MontoCapital.java
import java.util.Scanner;

public class MontoCapital {
    public static void main(String[] args) {
        Scanner entrada = new Scanner(System.in);
        double capital, interes, monto;

           System.out.println("Ingrese el capital: ");
           capital = entrada.nextDouble();
           System.out.println("Ingrese la tasa de interés: ");
           interes = entrada.nextDouble();

           monto = capital * (1 + (interes / 100));

           System.out.println("El monto es: " + monto);
     }
}
